project2/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+
+
+
+
+
+
+<body>
+
+
+
+
+
+    <div class="card">
+        <img src="Healthy-Garlic-Shrimp-Pasta-with-Lemon.jpg" alt="">
+
+        <h1>Garlic Shrimp Pasta
+        </h1>
+
+
+        <ul class="menu">
+            <li><a href="#fas fa-shopping-cart"><i class="fas fa-shopping-cart"></i></a></li>
```

Comment:
Your markup here is broken, as the link's `href` is blended with the classnames; instead, use something unique for the `id`s to make your links work:
```suggestion
            <li><a href="#ingredients"><i class="fas fa-shopping-cart"></i></a></li>
```
This is repeated on lines 34 and 35 below.